static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js

```diff
@@ -99,6 +70,44 @@ define([
                 }
                 return true;
             });
+
+            function open(isClosed_) {
+                var videoSrc = $('#YTPlayer').attr('src');
+                var videoSrcAutoplay = videoSrc;
+
+                if (videoSrc.indexOf('autoplay') === -1) {
+                    videoSrcAutoplay = videoSrc + '&amp;autoplay=1';
+                } else {
+                    videoSrcAutoplay = videoSrcAutoplay.replace(
+                        isClosed_ ? 'autoplay=0' : 'autoplay=1',
+                        isClosed_ ? 'autoplay=1' : 'autoplay=0'
+                    );
+                }
+
+                if (isClosed_) {
+                    $('.ad-exp__close-button', $adSlot[0]).addClass('button-spin');
+                    $('.ad-exp__open-chevron', $adSlot[0]).addClass('chevron-down');
+                    $ad.css('height', openedHeight);
+                    $fabricExpandableVideo.addClass('creative--open');
+                    $('.slide-video, .slide-video .ad-exp__layer', $adSlot[0])
+                    .css('height', openedHeight)
+                    .addClass('slide-video__expand');
+                } else {
+                    $('.ad-exp__close-button', $adSlot[0]).removeClass('button-spin');
+                    $('.ad-exp__open-chevron', $adSlot[0]).removeClass('chevron-down');
+                    $ad.css('height', closedHeight);
+                    $fabricExpandableVideo.removeClass('creative--open');
+                    $('.slide-video, .slide-video .ad-exp__layer', $adSlot[0])
+                    .css('height', closedHeight)
+                    .removeClass('slide-video__expand');
+                }
+
+                isClosed = !isClosed_;
```

Comment:
kinda confusing names for these two variables. One is a state, the other a transition